ros2topic/ros2topic/verb/bw.py

```diff
@@ -93,12 +96,20 @@ def __init__(self, node, window_size):
         self.sizes = []
         self.times = []
         self.window_size = window_size
+        self.use_sim_time = False
+        if node.has_parameter('use_sim_time'):
+            self.use_sim_time = node.get_parameter('use_sim_time')
```

Comment:
How about using `get_parameter_or` instead? alternative is `False`.

Reply:
It's OK, updated.